src/main/java/teammates/common/datatransfer/questions/FeedbackTextQuestionDetails.java

```diff
@@ -72,7 +72,7 @@ public String getQuestionWithoutExistingResponseSubmissionFormHtml(
             boolean sessionIsOpen, int qnIdx, int responseIdx, String courseId, int totalNumRecipients) {
         return Templates.populateTemplate(
                 FormTemplates.TEXT_SUBMISSION_FORM,
-                Slots.DISABLED, sessionIsOpen ? "" : "disabled",
+                Slots.IS_SESSION_CLOSED, sessionIsOpen ? "false" : "true",
```

Comment:
@HirdayGupta's suggestion reads slightly better, but we can live with the current one because it is still readable (and also to avoid the PMD violation)